js/repl/PluginConfig.js

```diff
@@ -116,6 +116,7 @@ const replDefaults: ReplState = {
   debug: false,
   evaluate: false,
   fileSize: false,
+  timeTravel: false,
```

Comment:
Can we leave a comment describing the difference between `timeTravel` and `showTimeTravel`?

Reply:
@existentialism removed `showTimeTravel` :)